deploy.pl

```diff
@@ -36,15 +37,33 @@
 
 my $ua = Mojo::UserAgent->new;
```

Comment:
well, I did not like it either, but this is from official documentation (http://mojolicious.org/perldoc/Mojo/UserAgent) so I thought "that's how they do it in perl" :-) but I will rename it